apps/fluent-tester/src/TestComponents/ColorTokens/ColorTokenTest.tsx

```diff
@@ -83,15 +88,74 @@ const AliasTokensSwatchList: React.FunctionComponent = () => {
     return Object.keys(aliasColorTokens).map(aggregator);
   }, [aliasColorTokens, aggregator]);
 
-  const aliasTokensAsArray = React.useMemo(flattenArray, [flattenArray]);
+  const aliasColorTokensAsArray = React.useMemo(flattenArray, [flattenArray]);
+
+  const renderSwatch = React.useCallback((item) => {
+    const { colorName, colorValue } = item;
+    return <ColorToken key={colorName} colorName={colorName} colorValue={colorValue} />;
+  }, []);
+
+  return (
+    <View style={commonTestStyles.view}>
+      <View style={styles.stackStyle}>{aliasColorTokensAsArray.map((item) => renderSwatch(item))}</View>
+    </View>
+  );
+};
+
+const globalNeutralColorNamesSortedDarkToLight = Object.keys(globalTokens.color)
+  .filter((globalColorName) => globalColorName.includes('grey') || globalColorName === 'black' || globalColorName === 'white')
+  .sort((color1, color2) => {
+    const color1hex = globalTokens.color[color1];
+    const color2hex = globalTokens.color[color2];
+    return color1hex === color2hex ? 0 : color1hex < color2hex ? -1 : 1;
+  });
+
+const globalNeutralColorTokensAsArray = globalNeutralColorNamesSortedDarkToLight.map((colorName: string) => {
+  return {
+    colorName: colorName,
+    colorValue: globalTokens.color[colorName],
+  };
+});
+
+const GlobalNeutralColorTokensSwatchList: React.FunctionComponent = () => {
   const renderSwatch = React.useCallback((item) => {
-    const { color, key } = item;
-    return <ColorToken key={key} color={color} name={key} />;
+    const { colorValue, colorName } = item;
+    return <ColorToken key={colorName} colorValue={colorValue} colorName={colorName} />;
   }, []);
+
   return (
     <View style={commonTestStyles.view}>
-      <Text>Alias Color Tokens from Token Pipeline</Text>
-      <View style={styles.stackStyle}>{aliasTokensAsArray.map((item) => renderSwatch(item))}</View>
+      <View style={styles.stackStyle}>{globalNeutralColorTokensAsArray.map((item) => renderSwatch(item))}</View>
+    </View>
+  );
+};
+
+const globalSharedColorNames = Object.keys(globalTokens.color).filter((key) => globalTokens.color[key].primary !== undefined);
+const globalSharedColorVariantsSortedDarkToLight = Object.keys(globalTokens.color.red).sort((color1, color2) => {
+  const color1hex = globalTokens.color.red[color1];
+  const color2hex = globalTokens.color.red[color2];
+  return color1hex === color2hex ? 0 : color1hex < color2hex ? -1 : 1;
+});
+const globalSharedColorTokensAsArray = globalSharedColorNames
+  .map((sharedColorName) => {
+    return globalSharedColorVariantsSortedDarkToLight.map((variant) => {
+      return {
+        colorName: sharedColorName + '.' + variant,
+        colorValue: globalTokens.color[sharedColorName][variant],
+      };
+    });
+  })
+  .reduce((accumulator, sharedColor) => accumulator.concat(sharedColor));
+
+const GlobalSharedColorTokensSwatchList: React.FunctionComponent = () => {
+  const renderSwatch = React.useCallback((item) => {
```

Comment:
Seems like you could pull this out into a generic function, since the code is duplicated across several components and doesn't depend on anything but the input